workload/locustfile.py

```diff
@@ -220,15 +255,46 @@ def netcdf_to_zarr_single_granule_argo(self):
     def netcdf_to_zarr_single_granule_turbo(self):
         self._netcdf_to_zarr_single_granule(True)
 
-    @tag('netcdf-to-zarr', 'async', 'zarr', 'argo', 'memory', 'slow')
-    @task(1)
-    def netcdf_to_zarr_large_granule_argo(self):
-        self._netcdf_to_zarr_large_granule()
+    # @tag('new', 'async', 'concise', 'argo')
```

Comment:
Is this commented out code being left here for a reason, or should it be deleted?